src/tests/class_tests/openms/CMakeLists.txt

```diff
@@ -38,8 +38,16 @@ project("OpenMS_class_tests_openms")
 #------------------------------------------------------------------------------
 # Configure test file to get the TEST_DATA_PATH into the tests
 set(CF_OPENMS_TEST_DATA_PATH "${PROJECT_SOURCE_DIR}/data/")
-set (CONFIGURED_TEST_CONFIG_H "${PROJECT_BINARY_DIR}/include/OpenMS/test_config.h")
-configure_file(${PROJECT_SOURCE_DIR}/include/OpenMS/test_config.h.in ${CONFIGURED_TEST_CONFIG_H})
+set(CONFIGURED_TEST_CONFIG_H "${PROJECT_BINARY_DIR}/include/OpenMS/test_config.h")
+configure_file(
+  ${PROJECT_SOURCE_DIR}/include/OpenMS/test_config.h.in
+  ${CONFIGURED_TEST_CONFIG_H}
+)
+
+#------------------------------------------------------------------------------
+# Find zlib and Boost.Iostreams for gzip support in tests
+find_package(ZLIB   REQUIRED)
+find_package(Boost  REQUIRED COMPONENTS iostreams)
```

Comment:
do you need this?
CoinOr and Eigen, for example, seem to be present already without searching?